pkg/knuu/instance.go

```diff
@@ -45,21 +46,21 @@ func NewInstance(name string) (*Instance, error) {
 	}
 	// Create the instance
 	return &Instance{
-		name:               name,
-		k8sName:            k8sName,
-		imageName:          "",
-		state:              None,
-		instanceType:       BasicInstance,
-		portsTCP:           make([]int, 0),
-		portsUDP:           make([]int, 0),
-		command:            make([]string, 0),
-		args:               make([]string, 0),
-		env:                make(map[string]string),
-		volumes:            make([]*k8s.Volume, 0),
-		memoryRequest:      "",
-		memoryLimit:        "",
-		cpuRequest:         "",
-		serviceAccountName: "default",
+		name:          name,
+		k8sName:       k8sName,
+		imageName:     "",
+		state:         None,
+		instanceType:  BasicInstance,
+		portsTCP:      make([]int, 0),
+		portsUDP:      make([]int, 0),
+		command:       make([]string, 0),
+		args:          make([]string, 0),
+		env:           make(map[string]string),
+		volumes:       make([]*k8s.Volume, 0),
+		memoryRequest: "",
+		memoryLimit:   "",
+		cpuRequest:    "",
+		policyRules:   make([]rbacv1.PolicyRule, 0),
```

Comment:
just for curiosity, does it means that the `serviceAccountName` is under the `rbacv1.PolicyRule`?